Packs/Reco/Integrations/Reco/Reco.py

```diff
@@ -191,7 +191,7 @@ def get_alerts(
         try:
             response = self._http_request(
                 method="PUT",
-                url_suffix="/alert-inbox/table",
+                url_suffix="/policy-subsystem/alert-inbox/table",
```

Comment:
What is the effect of this change on backward compatibility?

Reply:
No backward compatibility - we changed it